### PR TITLE
adds cais_* plays to site.yml playbook

### PR DESCRIPTION
**Pull request by andriy-safe-ai**
Added `cais_*` play to `site.yml` playbook. Doing this will run the plays when the cluster is first deployed.

### CODE REVIEW

**Comment by steven-basart** [comment]
I'm not okay with approving this unless we test it.  Last time we had it in it broke adding in a new node so we removed it.

**Comment by steven-basart** [comment]
It's also missing one of the playbooks.

**Comment by andriy-safe-ai** [comment]
@steven-basart 

Which playbook are we missing?

File: playbooks/site.yml [hunk 1]
```diff
@@ -326,3 +326,18 @@
...
+
+- hosts: all
+  tasks:
+    - include_role:
+        name: cais-all
+
+- hosts: bastion,login
+  tasks:
+    - include_role:
+        name: cais-bastion-all
```
Comment (steven-basart) [inline playbooks/site.yml:338]: https://github.com/centerforaisafety/cerberus-cluster/blob/main/playbooks/cais_bastion_login.yml

The role was misspelled

**Comment by steven-basart** [comment]
> Which playbook are we missing?

The role was misspelled
https://github.com/centerforaisafety/cerberus-cluster/blob/main/playbooks/cais_bastion_login.yml

That's why I thought it was missing.

**Comment by andriy-safe-ai** [comment]
@steven-basart 

Did some testing. One of our plays keeps failing. 

```
- name: Install llvm
  vars: 
    package_name: 
      - llvm-toolset-7.0
    package_state: latest
    package_repo: "epel,ol7_developer_EPEL"
  include_role: 
    name: safe_yum
  ignore_errors: true
```

Although this package does exist it is not available through the repos we use. If we'd want this package we would have to use a different repo. This link provides instructions: https://www.softwarecollections.org/en/scls/rhscl/llvm-toolset-7.0/

!! Do we still need this package? We don't have it on production. I ran a `sudo yum list install | grep llvm` and I only got back a result for `llvm-private.x86_64` version 7 but this is only the llvm engine and not the entire toolset.

**Comment by andriy-safe-ai** [comment]
@steven-basart 

Friendly bump.

**Comment by steven-basart** [comment]
I don't know what happened to it but we should support it. Not sure why it's failing "now" like GCC 5. It is a common enough build target. 

I'm pretty confused because I know I used it early on after installing it that way.

**Comment by andriy-safe-ai** [comment]
@steven-basart 

Why only version 7?

**Comment by steven-basart** [comment]
Can add more it was just the minimum version needed for that project to build at that time.

**Comment by andriy-safe-ai** [comment]
@steven-basart 

I'm not sure how you were able to install this package. I don't see it installed on the cluster so I'm not sure we need this at the moment. When someone does we can revisit this package but I want to close this issue so I'm going to remove this play.

**Comment by steven-basart** [comment]
Feel free to edit the playbook in another pr

**Comment by steven-basart** [comment]
You removed it and then added it back? I'm guessing this is for the rebase?

**Comment by andriy-safe-ai** [comment]
@steven-basart 

Yes, the rebase.

**Comment by steven-basart** [comment]
Needs squash

**Review by steven-basart — APPROVED** [review]
Needs squash

**Comment by andriy-safe-ai** [comment]
@steven-basart 

Squashed. Merging.

**Pull request merged**